frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/EditTab.test.tsx

```diff
@@ -18,7 +18,7 @@ describe('EditTab', () => {
   it('should render DisplayChosenOption', async () => {
     renderEditTab();
     expect(
-      screen.getByRole('button', { name: textMock('ux_editor.options.option_remove_text') }),
+      await screen.findByText(textMock('ux_editor.modal_properties_code_list_custom_list')),
```

Comment:
Why does this test have to be awaited and not use `getByText`? If there is due to some data needed from queries, it is better to adapt the tests to test for the spinner being there when waiting and for the data being there when the required data is on the cache.

Reply:
thank you for spotting it!